.about.yml

```diff
@@ -0,0 +1,72 @@
+---
+# .about.yml project metadata
+#
+# Short name that acts as the project identifier (required)
+name: openFEC-web-app
+
+# Full proper name of the project (required)
+full_name: openFEC-web-app
```

Comment:
Is this what's visible on the dashboard? Do we want a different public-facing name? @emileighoutlaw ?